src/org/usfirst/frc/team2335/robot/Robot.java

```diff
@@ -14,9 +14,18 @@
 	//Deadzone
 	public static final double DEADZONE = 0.15;
 	
+	//Motor controller constants
```

Comment:
Are these the correct ports or are we changing them later?

Reply:
They are the correct ports